SecurityPkg/FvReportPei/FvReportPei.c

```diff
@@ -352,12 +372,17 @@ CheckStoredHashFv (
                       );
   if (!EFI_ERROR (Status) && (StoredHashFvPpi != NULL) && (StoredHashFvPpi->FvNumber > 0)) {
     HashInfo = GetHashInfo (StoredHashFvPpi, BootMode);
-    Status   = VerifyHashedFv (
+    // MU_CHANGE [START] - CodeQL change
+    if (HashInfo != NULL) {
+      Status = VerifyHashedFv (
                  HashInfo,
                  StoredHashFvPpi->FvInfo,
                  StoredHashFvPpi->FvNumber,
                  BootMode
                  );
+    }

```

Comment:
If HashInfo is NULL shouldn't we edit Status to be some form of error?  Currently we can have it null and it'll assume the verification was successful.